DEVELOPER_GUIDE.md

```diff
@@ -132,14 +132,59 @@ $ yarn osd clean
 
 ### Run OpenSearch
 
-OpenSearch Dashboards requires a running version of OpenSearch to connect to. In a separate terminal you can run the latest snapshot built using:
+OpenSearch Dashboards requires a running version of OpenSearch to connect to. You can choose to run OpenSearch locally yourself or point to an existing cluster.
+
+#### Run a local OpenSearch cluster 
+
+In a separate terminal you can run the latest snapshot built using:
 
 _(Linux, Windows, Darwin (MacOS) only - for others, you'll need to [set up using Docker](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/docs/docker-dev/docker-dev-setup-manual.md) or [run OpenSearch from a tarball](#alternative---run-opensearch-from-tarball) instead)_
 
 ```bash
 $ yarn opensearch snapshot
 ```
 
+#### Alternative - Connect to an external OpenSearch cluster
+
+Instead of running OpenSearch locally, you can point OpenSearch Dashboards to an existing OpenSearch cluster:
+
+1. Clone the security dashboards plugin inside your OpenSearch Dashboards plugins folder and bootstrap:
+```bash
+$ cd plugins
+$ git clone https://github.com/opensearch-project/security-dashboards-plugin.git
+$ cd ..
+$ yarn osd bootstrap --single-version=loose
+```
+2. Create a configuration directory outside your repository to avoid accidentally committing credentials. For example:
```

Comment:
FYI, I normally use `opensearch_dashboards.dev.yml` to keep local dev configuration, this file is git ignored. But seems there is no documentation for this.